internal/command/migrate_to_v2/migrate_to_v2.go

```diff
@@ -236,6 +236,9 @@ func NewV2PlatformMigrator(ctx context.Context, appName string) (V2PlatformMigra
 	if err != nil {
 		return nil, err
 	}
+	allocs = lo.Filter(allocs, func(alloc *api.AllocationStatus, _ int) bool {
+		return !alloc.Failed && alloc.LatestVersion
```

Comment:
I've no idea what "latest_version" means :) ... digging ⌛

Reply:
at a glance: it looks like web returns all running allocations, even those that should be destroyed but haven't yet.
LatestVersion is true if alloc.version == last release version on the app 
it's an edge case, but this should filter out weird allocations that just don't die.